test/api/test_history_contents.py

```diff
@@ -1,11 +1,10 @@
-from base import api
 import json
 
-from .helpers import TestsDatasets
-from .helpers import LibraryPopulator
-from .helpers import DatasetCollectionPopulator
-from requests import delete
-from requests import put
+from requests import delete, put
+
+from base import api
+
+from .helpers import DatasetCollectionPopulator, LibraryPopulator, TestsDatasets
```

Comment:
> I've changed some imports in other files to be relative, but this works only if they are in the same directory (or in subdirectories), you cannot go up one or more levels (i.e. use more than one dot).
> ...
> A `galaxy_test` or `test_helpers` module would probably be the best solution for that, but I'm not sure where these files should go: under `lib/galaxy/` or?

@jmchilton I've fixed the rest of import order errors in https://github.com/galaxyproject/galaxy/compare/dev...nsoranzo:flake8-import-order?expand=1 , except for the above-mentioned type of imports.
The files/directories imported are:

- test/api/workflow_format_2/converter.py
- test/base/
- test/functional/database_contexts.py
- test/unit/tools/test_toolbox.py (because of BaseToolBoxTestCase and SimplifiedToolBox classes)
- test/unit/tools_support.py
- test/unit/unittest_utils/

I could use some help fixing this, at least I'd like an opinion on the above cited question.